.gitignore

```diff
@@ -0,0 +1,21 @@
+# Binaries for programs and plugins
```

Comment:
Should have a license header.

Reply:
Added - https://github.com/hyperledger/aries-framework-go/compare/0004b9425b06eddb52484f8fbfd11ffcc0d126e7..ffab3ad151e63cb395efa46935e28bfa1849f589